translate/storage/pypo.py

```diff
@@ -35,7 +35,6 @@
 from translate.misc.deprecation import deprecated
 from translate.misc.multistring import multistring
 from translate.storage import base, pocommon, poparser
-from translate.storage.pocommon import encodingToUse
```

Comment:
If I am not wrong this is the last usage of `encodingToUse` so perhaps it can also removed from codebase.